### PR TITLE
Update french.xml 

### PR DESCRIPTION
**Pull request by fylip22**
Update french.xml

### CODE REVIEW

File: PowerEditor/installer/nativeLang/french.xml [hunk 1]
```diff
@@ -834,7 +838,7 @@
                     <Item id="6219" name="Clignotement :"/>
                     <Item id="6221" name="Vif"/><!-- Rapide -->
                     <Item id="6222" name="Zen"/><!-- Lent -->
-                    <Item id="6225" name="Activer Multi-édition (Ctrl+Clic/sélection)"/>
+                    <Item id="6225" name="Activer multi-édition (Ctrl + clic sur la sélection)"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/french.xml:841]: Please restore it by the same guideline.

File: PowerEditor/installer/nativeLang/french.xml [hunk 2]
```diff
@@ -907,22 +911,22 @@ Vous pouvez définir plusieurs marqueurs de colonne en utilisant un espace pour
                 </NewDoc>
 
                 <DefaultDir title="Dossier par défaut">
-                    <Item id="6413" name="Dossier par défaut pour ouvrir/enregistrer les fichiers"/>
+                    <Item id="6413" name="Dossier par défaut pour ouvrir ou enregistrer les fichiers"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/french.xml:914]: ditto above

File: PowerEditor/installer/nativeLang/french.xml [hunk 3]
```diff
@@ -1087,7 +1092,7 @@ Vous pouvez définir plusieurs marqueurs de colonne en utilisant un espace pour
                     <Item id="6313" name="Mise à jour sans alerte"/>
                     <Item id="6322" name="Ext. fichier session :"/>
                     <Item id="6323" name="Activer la mise à jour automatique"/>
-                    <Item id="6324" name="Commutateur de documents (Ctrl+Tab)"/>
+                    <Item id="6324" name="Commutateur de documents (Ctrl + Tab)"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/french.xml:1095]: ditto above (Restore it please)

File: PowerEditor/installer/nativeLang/french.xml [hunk 4]
```diff
@@ -802,11 +805,11 @@
...
-                    <Item id="6103" name="Fluent UI: petites icônes"/>
-                    <Item id="6104" name="Fluent UI: grandes icônes"/>
-                    <Item id="6129" name="Filled Fluent UI: petites icônes"/>
-                    <Item id="6130" name="Filled Fluent UI: grandes icônes"/>
-                    <Item id="6105" name="Standards: petites icônes"/>
+                    <Item id="6103" name="Icônes ajourés : petite taille"/>
+                    <Item id="6104" name="Icônes ajourés : grande taille"/>
+                    <Item id="6129" name="Icônes pleins : petite taille"/>
+                    <Item id="6130" name="Icônes pleins : grande taille"/>
+                    <Item id="6105" name="Standard : petites icônes"/>
```
[the remark this answers is not in the document]
Comment (Mateos81) [reply PowerEditor/installer/nativeLang/french.xml:812]: I'd also point out that the 1st 3 lines are masculine while the 4th one if feminine; would be better
And it should be feminine, see https://fr.wikipedia.org/wiki/Ic%C3%B4ne_(informatique)#cite_note-1

[the remark this answers is not in the document]
Comment (donho) [reply PowerEditor/installer/nativeLang/french.xml:812]: OK, let's make it feminine.

**Comment by donho** [comment]
@fylip22 
Could you adapt this PR to meet the request please?

**Comment by donho** [comment]
Due to no reply from the author of PR, this PR is closed.

**Pull request closed without merging**